src/gui/entry/EntryAttributesModel.cpp

```diff
@@ -19,6 +19,18 @@
 
 #include "core/EntryAttributes.h"
 
+#include <QCollator>
+
+namespace
+{
+    void sortLocaleAware(QList<QString>& strings)
+    {
+        QCollator collator;
```

Comment:
Need to instantiste QCollator once, it's a very expensive (time wise) class to create

Reply:
Thanks for the feedback!

Considering that this PR is from 2023 and I haven't worked on it since then: Do you think it is worth it to continue with this PR? I remember that I opened the PR, however some CI pipeline failed and I put it back to draft mode. Haven't had a look into it since then.

I could try to rebase the branch onto dev and implement your feedback if you think the PR is worth it, otherwise I could withdraw it given the long inactivity time.

Reply:
I can handle the rebase.